gold-zip-input.html

```diff
@@ -140,8 +141,20 @@
             this.$.input.selectionStart = start + 1;
             this.$.input.selectionEnd = start + 1;
           }
-        }
+        },
 
+        /**
+         * Recreates a change event that's targetted against this element if it
+         *  doesn't have a shadowRoot, turns out that's handled for us if it does.
+         */
+        __onChange: function() {
```

Comment:
Actually, why do you need this at all? Doesn't it come from PaperInputBehavior, where it's already implemented?